src/pybind/mgr/dashboard/services/access_control.py

```diff
@@ -195,7 +199,12 @@ def enabled(self, value):
         self._enabled = value
         self.refreshLastUpdate()
 
-    def set_password(self, password):
+    @staticmethod
+    def update_lastPwdUpdate(lastPwdUpdate):
```

Comment:
```suggestion
    def update_last_pwd_update(last_pwd_update):
```

Consider using a different method name, e.g. ``set_last_pwd_update``.